astropy/tests/__init__.py

```diff
@@ -4,8 +4,3 @@
 for writing tests, and general tests that are not associated with a
 particular package.
 """
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
-
-from . import helper
-from .disable_internet import turn_off_internet, turn_on_internet
```

Comment:
No, it's never been documented as such.  I don't think this import being here was ever particularly deliberate, and nobody seems to rely on it.